package.json

```diff
@@ -44,13 +44,19 @@
   ],
   "scripts": {
     "toc": "doctoc README.md",
-    "lint": "(prettier . --check || true) && eslint .",
+    "lint": "prettier . --check && eslint .",
+    "lint:delta": "npm-run-all -p prettier:delta eslint:delta",
+    "prettier:delta": "prettier --check `./scripts/changed-files`",
+    "eslint:delta": "eslint `./scripts/changed-files`",
     "format": "prettier . --write && eslint . --fix",
+    "format:delta": "npm-run-all format:prettier:delta format:eslint:delta",
+    "format:prettier:delta": "prettier --write `./scripts/changed-files`",
+    "format:eslint:delta": "eslint --fix `./scripts/changed-files`",
```

Comment:
- POSIX recommends `$(...)` over backticks for shell command substitution, IIRC
- I don't think this is Windows friendly
- Currently, if the bash script fails, the npm script will still run, causing the failure of the script to be swallowed in CI

<img width="696" alt="Screenshot 2024-01-30 at 05 30 25" src="https://github.com/testing-library/svelte-testing-library/assets/2963448/afb41bf3-6891-42e6-94a6-04a91ae0ab14">